src/models/tag/stream.rs

```diff
@@ -22,6 +20,47 @@ pub enum TagStreamReach {
     Followers,
     Following,
     Friends,
+    MostFollowed,
+    Influencers,
+}
+
+impl TagStreamReach {
+    pub fn to_graph_subquery(&self) -> String {
+        let query = match self {
+            TagStreamReach::Followers => "MATCH (user:User)<-[:FOLLOWS]-(reach:User)",
+            TagStreamReach::Following => "MATCH (user:User)-[:FOLLOWS]->(reach:User)",
+            TagStreamReach::Friends => {
+                "MATCH (user:User)-[:FOLLOWS]->(reach:User), (user)<-[:FOLLOWS]-(reach)"
+            }
+            TagStreamReach::MostFollowed => "",
+            TagStreamReach::Influencers => "",
+        };
+        String::from(query)
+    }
+}
+
+#[derive(Deserialize, Debug, ToSchema, Clone)]
+pub enum TaggedType {
+    Post,
+    User,
+}
```

Comment:
It's not inmediately obvious to me whether we need really need this or whether this is the place for it. Maybe we do, maybe we don't? Certainly on `Post` is needed in current design.

Reply:
It's providing what the frontend team needs right now, and will also support future use-cases